### DISCUSSION TITLE
I cant submit icons :/

### DISCUSSION THREAD

**Discussion by atomtoto** [elrumo/macOS_Big_Sur_icons_replacements#1145] [Help with the website] [answered]
Whenever i want to send icons, it remains blocked on the loading page... What to do ?
[https://nsa40.casimages.com/img/2021/04/27/210427034243811357.png](url)

**Comment by cake03** [comment]
@atom-2jong, Which web browser do you use?

**Reply by atomtoto** [reply]
Firefox on MacOS Big Sur, and also on Safari on iOS 14.6

**Reply by atomtoto** [reply]
Maby it's the cookies and add blocker ?

**Comment by elrumo** [comment, marked as answer]
How did you export the icons? What tool did you use?

**Reply by atomtoto** [reply]
I try to export my icons from your website macOSicons. But its dont working. I don't know why 🤷🏻. I try wise a iPhone SE from safari and also wise Firefox a iMac. Thanks for responding me. ( sorry my English is veeeeerryyy bad lol)

**Reply by atomtoto** [reply]
I make them wise Figma online and I export it in png

**Reply by elrumo** [reply]
Can you share a screenshot of what the export section looks like? 
Like this: ![image](https://user-images.githubusercontent.com/36299870/116560133-00e0fd00-a8f9-11eb-81cc-fe1927bad745.png)

**Reply by atomtoto** [reply]
![image](https://user-images.githubusercontent.com/81765688/116570737-c7b18880-a90a-11eb-8bb2-4432450643d7.jpeg)

**Reply by elrumo** [reply]
> ![image](https://user-images.githubusercontent.com/81765688/116570737-c7b18880-a90a-11eb-8bb2-4432450643d7.jpeg)

I think the issue may be that you're exporting at 4x, make sure you're exporting at 1x and the final file has a resolution of 1024x1024

**Reply by atomtoto** [reply]
Thanks man !!!

**Reply by atomtoto** [reply]
I submit it 😉
